theme/layouts/_header.ejs

```diff
@@ -0,0 +1,17 @@
+<header>
+    <div id="header-top" class="col2 device">
+      <a href="/" id="header-logo" class="cell1 start">
+        <img src="<%= site.root %>/images/logo.png" alt="Privacy Manager logo" />
+        <p class="tablet-hide">
+          <%= site.title %>
+        </p>
+      </a>
+      <button id="hamburger" class="cell2 end desktop-hide" aria-label="Open primary navigation"></button>
+      <div class="cell2 end tablet-hide">
+        <% include _language %>
+      </div>
+    </div>
+    <div id="hamburger-menu" class="col1">
```

Comment:
Detail: This is no more `#hamburger-menu` is you are using it for generic menu as well.

Reply:
Not even sure if you also need ID at all, as you can use `header nav`? Because we only have 1 header navigation.

Reply:
Done.